packages/@react-aria/numberfield/src/useNumberField.ts

```diff
@@ -175,7 +181,12 @@ export function useNumberField(props: AriaNumberFieldProps, state: NumberFieldSt
     id: inputId,
     type: 'text', // Can't use type="number" because then we can't have things like $ in the field.
     inputMode,
-    onChange
```

Comment:
spread didn't work, typescript error, this was a suggested possible solution that it worked.